numberformat/diff.emu

```diff
@@ -1091,6 +1091,11 @@
         1. Let _literal_ be ParseText(_text_, |StringNumericLiteral|).
         1. If _literal_ is a List of errors, let _mv_ be ~not-a-number~.
         1. Else, let _mv_ be the StringIntlMV of _literal_.
+        1. If _mv_ ≥ 1<sup>1000</sup>, then let _mv_ be ~positive-infinity~.
```

Comment:
My understanding is for the &lt;emu-alg&gt; any "If xxxx , then" should have substeps and if you want to have the same line If, it should not have ", then" so you should remove the "then" before the "let" next 4 lines.